posthog/models/person/util.py

```diff
@@ -162,6 +162,26 @@ def get_persons_by_uuids(team: Team, uuids: List[str]) -> QuerySet:
 
 
 def delete_person(person: Person) -> None:
+    def _get_distinct_ids_with_version() -> Dict[str, int]:
+        return {
+            distinct_id: int(version or 0)
+            for distinct_id, version in PersonDistinctId.objects.filter(person=person, team_id=person.team_id)
+            .order_by("id")
+            .values_list("distinct_id", "version")
+        }
+
+    def _delete_ch_distinct_ids(distinct_ids_to_version: Dict[str, int]):
```

Comment:
It's uncommon to create nested functions like this in python. Could we move this to the top level?